cob_scan_unifier/src/scan_unifier_node.cpp

```diff
@@ -28,6 +28,7 @@ ScanUnifierNode::ScanUnifierNode()
 
   // Publisher
   topicPub_LaserUnified_ = nh_.advertise<sensor_msgs::LaserScan>("scan_unified", 1);
+  topicPub_PointcloudUnified_ = nh_.advertise<sensor_msgs::PointCloud2>("/pointcloud_unified", 1);
```

Comment:
use the same namespace as for scan_unified
```suggestion
  topicPub_PointcloudUnified_ = nh_.advertise<sensor_msgs::PointCloud2>("pointcloud_unified", 1);
```